app/models/mixins/assignment_mixin.rb

```diff
@@ -81,7 +85,13 @@ def get_assigned_tos
         tag = Tag.find_by(:name => "/" + parts.join("/"))
         result[:tags] << [Classification.find_by(:tag_id => tag.id), klass] unless tag.nil?
       when :label
-        label = CustomAttribute.find_by(:name => parts[1], :value => parts[2])
+        label = if parts[1].starts_with?(BASE64_PREFIX)
+                  name = Base64.decode64(parts[1].slice(BASE64_PREFIX.length+2..-2))
+                  value = Base64.decode64(parts[2].slice(BASE64_PREFIX.length+2..-2))
+                  CustomAttribute.find_by(:name => name, :value => value)
+                else
+                  CustomAttribute.find_by(:name => parts[1], :value => parts[2])
+                end
         result[:labels] << [label, klass] unless label.nil?
       end
```

Comment:
We are loading assigned labels from the db to the chargeback editor. We need to figure out if we are workign with a new base64 format or the old regular one.